README.md

```diff
@@ -147,6 +147,23 @@ The following platforms are currently not supported with the gProfiler executabl
 
 **Remark:** container-based execution works and can be used in those cases.
 
+## Running as systemd service
+
+You can generate a systemd service configuration that [runs gProfiler as an executable](#running-as-an-executable) (and therefore, bears the same [known issues](#executable-known-issues)) by running:
+
+``` bash
+GPROFILER_VERSION=<VERSION> GPROFILER_TOKEN=<TOKEN> GPROFILER_SERVICE=<SERVICE_NAME> ./deploy/systemd/create_systemd_service.sh
```

Comment:
I don't necessarily want people to clone the repo for it - so let's give a link to the raw githubcontent site for this file (from master) so that the user can easily download